src/Frame.js

```diff
@@ -0,0 +1,51 @@
+function Frame(finalFrame) {
+  this._rolls = [];
+  this._bonus = null;
```

Comment:
Watch out for having state that can have a value that would cause an error.  eg in this case, what happens if someone uses `_bonus` when it's still `null`?